src/simple_lottery.rs

```diff
@@ -0,0 +1,185 @@
+//! ALBA Telescope with simple lottery construction given in Section 4.1.
+
+/// Setup output parameters
+#[derive(Debug, Clone)]
+pub struct LotterySetup {
+    /// Proof size (in Sp elements)
+    pub u: u64,
```

Comment:
I thought we want to have these parameters with more descriptive names in the code - e.g., `u --> proof_size`?